.eslintrc.json

```diff
@@ -1,17 +1,21 @@
 {
   "extends": [
-    "okonet"
+    "okonet",
+    "plugin:node/recommended"
   ],
+  "plugins": ["node"],
   "env": {
     "browser": false,
     "node": true,
     "jest": true,
     "es6": true
   },
   "rules": {
-    "strict": 0,
-    "import/no-extraneous-dependencies": [2, {
-      "devDependencies": ["**/*.spec.js"]
-    }]
+    "strict": "off",
+    "no-console": "off",
+    "node/exports-style": [
```

Comment:
Could you turn this off?

Reply:
This is a good enforcement, right? To prevent

```js
module.exports = {
    foo: 1
}

exports.bar = 2
```

Reply:
Having this rule does not allow to use the [`exports` shortcut](https://nodejs.org/api/modules.html#modules_exports_shortcut). While the example above is a valid reason for having the rule, I think it is mostly not relevant for us. To begin with, the likelihood of any of us writing such buggy code is low. Then we have tests which will most definitely fail. And if even that doesn't happen, it will be caught during code review.

Reply:
oh sorry, I didn't see this 😢 
Okay, I'll update this PR

Reply:
No problem at all! And thanks!